docs/warnings.rst

```diff
@@ -42,7 +42,7 @@ using the `~warnings.catch_warnings` context manager::
 
     >>> from warnings import catch_warnings
     >>> with catch_warnings():
-    ...     warnings.filterwarnings('ignore', AstropyUserWarning)
+    ...     warnings.filterwarnings('ignore', category=AstropyUserWarning)
```

Comment:
Perhaps a better spelling would be:
```python
warnings.simplefilter('ignore', AstropyUserWarning)
```
In this case the warning category as the second argument is correct, and is probably what the original example intended to say.  I'm fine either way, but sort of prefer the simplefilter spelling just since it's a bit more terse?  UP to you.